src/globus_cli/commands/timer/create/transfer.py

```diff
@@ -174,6 +186,17 @@ def transfer_command(
     source_endpoint, cmd_source_path = source
     dest_endpoint, cmd_dest_path = destination
 
+    if delete:
+        click.echo(
+            click.style(
+                "`--delete` has been deprecated and "
+                "will be removed in a future release. "
+                "Use --delete-destination-extra instead.",
+                fg="yellow",
+            ),
+            err=True,
+        )
```

Comment:
The implicit string concatenation, which looks like three separate positional arguments due to whitespace alignment with the keyword argument `fg`, is not ideal. I recommend a formulation like this:

```suggestion
        msg = (
            "`--delete` has been deprecated and will be removed in a future release. "
            "Use --delete-destination-extra instead."
        )
        click.echo(click.style(msg, fg="yellow"), err=True)
```